WebClient/Content/WindTemperatureView.css

```diff
@@ -0,0 +1,3 @@
+.d3-control-windTemperatureView {
```

Comment:
WindTemperature is Wind + Temperature. What if we will add humidity in the future. We need a more generic name here, e.g. WeatherControl => weatherControlView.css

Reply:
Done